pkg/okteto/auth.go

```diff
@@ -206,6 +210,16 @@ func GetUsername() string {
 	return t.Username
 }
 
+// GetSanitizedUsername returns the username of the authenticated user sanitized to be DNS compatible
+func GetSanitizedUsername() string {
```

Comment:
what do we need this for? Should we instead get the API to return this in a kube-friendly way? I feel like this is server-side behavior that the CLI shouldn't have to know about.

Reply:
if not, is there kube library we can use for this? I think there are other rules we might be missing here (e.g. using dashes, underscores, or that a name can't end on an underscore)